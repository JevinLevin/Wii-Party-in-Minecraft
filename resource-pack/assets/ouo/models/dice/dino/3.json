{
	"credit": "Made by OuOuo Workshop, and big thx to Jannis!",
	"parent": "ouo:dice/parent/3",
	"textures": {
		"1": "ouo:dice/dino",
		"particle": "ouo:dice/dino"
	}
}