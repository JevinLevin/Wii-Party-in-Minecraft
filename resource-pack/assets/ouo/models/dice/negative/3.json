{
	"credit": "Made by OuOuo Workshop, and big thx to Jannis!",
	"parent": "ouo:dice/parent/3",
	"textures": {
		"1": "ouo:dice/negative",
		"particle": "ouo:dice/negative"
	}
}