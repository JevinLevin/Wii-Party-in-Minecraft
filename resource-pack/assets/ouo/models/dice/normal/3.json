{
	"credit": "Made by OuOuo Workshop, and big thx to Jannis!",
	"parent": "ouo:dice/parent/3",
	"textures": {
		"1": "ouo:dice/white",
		"particle": "ouo:dice/white"
	}
}