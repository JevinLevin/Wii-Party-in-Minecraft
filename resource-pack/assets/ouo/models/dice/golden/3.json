{
	"credit": "Made by OuOuo Workshop, and big thx to Jannis!",
	"parent": "ouo:dice/parent/3",
	"textures": {
		"1": "ouo:dice/golden",
		"particle": "ouo:dice/golden"
	}
}